Misc/NEWS.d/next/Library/2024-07-03-10-11-53.gh-issue-121313.D7gARW.rst

```diff
@@ -0,0 +1 @@
+Limit the reading size in the :class:`multiprocessing.Connection` class to 64KiB to prevent memory overallocation and unnecessary memory management system calls.
```

Comment:
```suggestion
Limit the reading size in the :class:`multiprocessing.Connection` class to 64 KiB to prevent memory overallocation and unnecessary memory management system calls.
```